common/src/chain/config/mod.rs

```diff
@@ -525,6 +526,70 @@ fn create_testnet_genesis() -> Genesis {
     )
 }
 
+pub fn regtest_genesis_values() -> (
+    PoolId,
+    Box<StakePoolData>,
+    PrivateKey,
+    PublicKey,
+    VRFPrivateKey,
+    VRFPublicKey,
+) {
+    let genesis_pool_id =
+        decode_hex::<PoolId>("123c4c600097c513e088b9be62069f0c74c7671c523c8e3469a1c3f14b7ea2c4");
+
+    let genesis_stake_private_key = decode_hex::<PrivateKey>(
+        "008717e6946febd3a33ccdc3f3a27629ec80c33461c33a0fc56b4836fcedd26638",
+    );
+
+    let genesis_stake_public_key = decode_hex::<PublicKey>(
+        "0003c53526caf73cd990148e127cb57249a5e266d78df23968642c976a532197fdaa",
+    );
+
+    let genesis_vrf_private_key = decode_hex::<VRFPrivateKey>("003fcf7b813bec2a293f574b842988895278b396dd72471de2583b242097a59f06e9f3cd7b78d45750afd17292031373fddb5e7a8090db51221038f5e05f29998e");
+
+    let genesis_vrf_public_key = decode_hex::<VRFPublicKey>(
+        "00fa2f59dc7a7e176058e4f2d155cfa03ee007340e0285447892158823d332f744",
+    );
+
+    let genesis_pool_stake_data = Box::new(StakePoolData::new(
+        MIN_STAKE_POOL_PLEDGE,
+        Destination::PublicKey(genesis_stake_public_key.clone()),
+        genesis_vrf_public_key.clone(),
+        Destination::PublicKey(genesis_stake_public_key.clone()),
+        PerThousand::new(1000).expect("Valid per thousand"),
+        Amount::ZERO,
+    ));
+
+    (
+        genesis_pool_id,
+        genesis_pool_stake_data,
+        genesis_stake_private_key,
+        genesis_stake_public_key,
+        genesis_vrf_private_key,
+        genesis_vrf_public_key,
+    )
+}
+
+pub fn create_regtest_pos_genesis() -> Genesis {
+    let (
+        genesis_pool_id,
+        genesis_stake_pool_data,
+        _genesis_stake_private_key,
+        _genesis_stake_public_key,
+        _genesis_vrf_private_key,
+        _genesis_vrf_public_key,
+    ) = regtest_genesis_values();
+
+    let create_genesis_pool_txoutput =
+        TxOutput::CreateStakePool(genesis_pool_id, genesis_stake_pool_data);
+
+    Genesis::new(
+        String::new(),
+        BlockTimestamp::from_int_seconds(1639975460),
+        vec![create_genesis_pool_txoutput],
```

Comment:
I'd still create some coins because it might be useful for delegation

Reply:
Ah, good point. Thanks will do.